pkg/log/log.go

```diff
@@ -165,7 +165,8 @@ func format(msg, level string, ts time.Time, appInfo Marshaler, mm Many) string
 
 	var b bytes.Buffer
 	if err := json.NewEncoder(&b).Encode(entry); err != nil {
-		log.Fatal(err)
+		log.Printf("could not encode %v; %s", entry, err.Error())
```

Comment:
Perhaps, instead of calling the standard logger, it would make more sense to just write to `b` with the raw contents of the entry? 

Also, I know this is nitpicky, but it would still be hard to determine where this is coming from right now with this change. I think we should denote that somewhere in the log message.

Reply:
Sounds reasonable. Just so I’m clear, do you mean the results of calling fmt.Sprintf(%v)?

Reply:
Something like `b.WriteString(fmt.Sprint(entry))`, but with my last comment more like:

```go
b.WriteString(fmt.Sprintf("gobox/log: failed to json encode log entry %v; err=%v", entry, err))
```

Reply:
It's also _highly_ possible that `%v` will be almost useless here, so we _may_ end up wanting to do something like `spew.Sdump(entry)` instead... but we can figure that out after this goes out IMO.

Reply:
I would also highly recommend adding the type of the entry to whatever log printed, e.g.
`fmt.Sprintf("gobox/log: failed to json encode log entry %v of type %T; err=%v", entry, entry, err`

Reply:
We could do an expensive call for a stack trace and write it into the buffer?

Reply:
yeah, i think i'll leave out the stack, since that seems to have some implications i dont really want to think about right now, and at some point we're gonna collide with the logging rework effort

Reply:
done.